src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/ViewModels/DataSetViewModels.cs

```diff
@@ -0,0 +1,124 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Converters.SystemJson;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using Newtonsoft.Json.Converters;
+using System.Net.NetworkInformation;
+using System.Security.Policy;
+using System.Text.Json.Serialization;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels;
+
+public record DataSetViewModel
+{
+    /// <summary>
+    /// The ID of the data set. 
+    /// <br/><br/>
+    /// thing
+    /// </summary>
+    public required Guid Id { get; init; }
+
+    /// <summary>
+    /// The title of the data set.
+    /// </summary>
+    public required string Title { get; init; }
+
+    /// <summary>
+    /// A summary of the data set’s contents.
+    /// </summary>
+    public required string Summary { get; init; }
+
+    /// <summary>
+    /// The status of the data set. Can be one of the following: 
+    /// <br/><br/>
+    /// Published - the data set has been published and will receive updates \
+    /// Deprecated - the data set is being discontinued and will no receive updates \
+    /// Unpublished - the data set has been unpublished and can no longer be used 
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public required DataSetStatus Status { get; init; }
```

Comment:
As mentioned in Slack, we probably need to filter out `Staged` as a enum value in the Swagger docs.

[This commit](https://github.com/dfe-analytical-services/explore-education-statistics/commit/155590451240f6e693467b908058ef9cbbbdd376) on my data set query work illustrates how we can modify individual schemas.

Note that I've done it this way for `GeographicLevel` and `TimeIdentifier` as they both live in Common. Consequently, it gets really messy to have to apply annotations and docblocks that apply correctly in all context. 

We could alternatively copy the enums over into the public API project, but felt icky given how much logic is tied into both, and how many enums there are (particularly for `TimeIdentifier`).